### PR TITLE
Update Readme to say what repo contains

### PR DESCRIPTION
**Pull request by EoinShaughnessy**
Fixes [#1287](https://github.com/alphagov/govuk-design-system/issues/1287).

PR raised because [users have pointed out that the Readme currently lacks info about the repo's purpose](https://github.com/alphagov/govuk-frontend/issues/1833#issuecomment-642681664).

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,7 +1,6 @@
 # GOV.UK Design System
 
-**One place for service teams to find styles, components and patterns for
-designing government services.**
+This repository contains the code for the GOV.UK Design System website. To find the code we provide for reuse by services, go to the [govuk-frontend repository](https://github.com/alphagov/govuk-frontend).
```
Comment (EoinShaughnessy) [inline README.md:3]:
```suggestion
This repository contains the code for the GOV.UK Design System website. For example, its documentation and visual styles. To find the code we provide for reuse by services, go to the [govuk-frontend repository](https://github.com/alphagov/govuk-frontend).
```

**Review by 36degrees — APPROVED** [review]
Looks good to me! Thanks @EoinShaughnessy 👍🏻

**Comment by EoinShaughnessy** [comment]
@36degrees Thanks for reviewing!

File: README.md [hunk 2]
```diff
@@ -1,7 +1,6 @@
 # GOV.UK Design System
 
-**One place for service teams to find styles, components and patterns for
-designing government services.**
+This repository contains the code for the GOV.UK Design System website. For example, for its documentation and visual styles. To find the code we provide for reuse by services, go to the [govuk-frontend repository](https://github.com/alphagov/govuk-frontend).
```
Comment (EoinShaughnessy) [inline README.md:3]:
```suggestion
This repository contains the code for the GOV.UK Design System website. To find the code we provide for reuse by services, go to the [govuk-frontend repository](https://github.com/alphagov/govuk-frontend).
```

**Pull request merged**